src/rp2_common/boot_bootrom_headers/BUILD.bazel

```diff
@@ -0,0 +1,10 @@
+package(default_visibility = ["//visibility:public"])
+
+cc_library(
+    name = "boot_bootrom_headers",
```

Comment:
I suspect there may be a circular dependency if you try to add this to the deps of [//src/rp2_common/pico_bootrom:pico_bootrom_headers](https://github.com/raspberrypi/pico-sdk/blob/efe2103f9b28458a1615ff096054479743ade236/src/rp2_common/pico_bootrom/BUILD.bazel#L17), so I'm game to sort out the complexity here in a follow-up.

Reply:
thanks - yes figured this would break u guys (though I didn't know about the circular dependency - i just copied the dependency on pico_platform_internal - not sure if it is needed